CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
```

Comment:
@galwiner please add the version at the bottom of this file as well

Reply:
done